docs/cookbook/configuration.rst

```diff
@@ -12,7 +12,7 @@ You can use a different config file name and path with the ``--config`` option:
     $ bin/phpspec run --config path/to/different-phpspec.yml
 
 You can also specify default values for config variables across all repositories by creating
-the file ``.phpspec.yml`` in your home folder (Unix systems). Phpspec will use your personal preference for
+the file ``phpspec.yml`` in your home folder (Unix systems). Phpspec will use your personal preference for
```

Comment:
This is wrong. PhpSpec does use `.phpspec.yml` from your home directory. It looks for `phpspec.yml` and `phpspec.yml.dist` in your project directory.